CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Node: Added ZRANGE command ([#1115](https://github.com/aws/glide-for-redis/pull/1115))
 * Python: Added RENAME command ([#1252](https://github.com/aws/glide-for-redis/pull/1252))
 * Python: Added APPEND command ([#1152](https://github.com/aws/glide-for-redis/pull/1152))
+* Python: Add `PFADD` command (TODO PR #)
```

Comment:
not a big deal but I'd say lets follow the existing format
```suggestion
* Python: Added PFADD command (TODO PR #)
```

Reply:
Should be there a feature name or PR title?

Reply:
Took a look at their PR titles and they don't necessarily match the CHANGELOG entry exactly. I'd match the existing format here though, and then the PR title can either be the same or slightly different.